README.md

```diff
@@ -136,11 +136,22 @@ See [examples on how to use opengpt with different models.](./examples) 🔥
 To do so, you can use the `serve` command:
 
 ```bash
-opengpt serve stabilityai/stablelm-tuned-alpha-3b --precision fp16 --device_map balanced
+opengpt serve yahma/llama-7b-hf --precision fp16 --device_map balanced
 ```
 
+or 
+
+```bash
+opengpt serve yahma/llama-7b-hf --adapter jinaai/alpaca-lora --precision bit8 --device_map balanced
+```
+
+if you want to use adapter to optimize VRAM usage.
+
 💡 **Tip**: you can inspect the available options with `opengpt serve --help`.
 
+> **Note**
+> `adapter` is only available for `LLaMA` model for now.
+
```

Comment:
To highlight this functionality, we should mention it at the beginning of README. Maybe we can raise a new PR to update adaptor supporting in our readme.